inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -177,4 +181,93 @@ public final class Constants {
                     .defaultValue(false)
                     .withDescription("Because spark do not support iceberg data type: `timestamp without time zone` and"
                             + "`time`, so type conversions must be mapped to types supported by spark.");
+
+    //========================================= dirty configuration =========================================
+    public static final String DEFAULT_LABEL_IDENTIFER = "dirty-data";
+    public static final String SINK_DIRTY_PREFIX = "sink.dirty.";
+    public static final String DIRTY_PREFIX = "dirty.";
+    public static final String DIRTY_LABEL_DELIMITER = "-";
+
+    public static final ConfigOption<Boolean> DIRTY_IGNORE =
```

Comment:
what is the difference between ignoring dirty data and ignoring sink errors? should sink error only occur when there is dirty data that is not being ignored?

Reply:
It has stated in the description.